### PR TITLE
Getting freetype2 from meson wrap

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
Currently freetype2 is get using Nuget, but as freetype2 do exists as a meson wrap, we could be using it to built from sources instead of getting binaries with Nuget.

### CODE REVIEW

**Comment by JPTIZ** [comment]
Why did we stop this exactly?

**Comment by joaoantoniocardoso** [comment]
> Why did we stop this exactly?

Actually I don't remember very well, I think at the time it was only building in my machine for unknown reasons... Maybe something related with the recent patches about pkgconfig?

File: .gitignore [hunk 1]
```diff
@@ -100,8 +100,5 @@ efl_libs.csv
 env.bat
 
 # Subprojects
-subprojects/pcre-*
-subprojects/zlib-*
-subprojects/getopt
-subprojects/packagecache
-subprojects/check*
+subprojects/*
+!subprojects/*.wrap
```
Comment (JPTIZ) [inline .gitignore:104]: You're a genious!

**Comment by joaoantoniocardoso** [comment]
@JPTIZ Note that the dependency called `freetype` is not equivalent to `freetype2`. The dependency `freetype` is located at `src/static_libs/freetype` and those who wants `freetype` is requesting headers that doesn't exist in `freetype2`.

**Comment by JPTIZ** [comment]
> @JPTIZ Note that the dependency called `freetype` is not equivalent to `freetype2`. The dependency `freetype` is located at `src/static_libs/freetype` and those who wants `freetype` is requesting headers that doesn't exist in `freetype2`.

Now things do make sense, thanks haha

File: header_checks/meson.build [hunk 2]
```diff
@@ -219,6 +219,9 @@ endif
 # libcheck
 check = dependency('check', fallback : ['check', 'check_dep'])
 
+# freetype2
+freetype2 = dependency('freetype2', fallback : ['freetype2', 'freetype_dep'])
```
Comment (joaoantoniocardoso) [inline header_checks/meson.build:223]: shouldn't it be `freetype2_dep`?

Comment (JPTIZ) [reply header_checks/meson.build:223]: If you're saying it should be `freetype2_dep = [...]`: nope, `freetype2` is the variable used around EFL (while `freetype2_dep` has no results). 

If you're saying it should be `['freetype2', 'freetype2_dep']`: also nope, [see the wrap's meson variable](https://github.com/mesonbuild/freetype2/blob/2.9.1/meson.build#L227).

Comment (joaoantoniocardoso) [reply header_checks/meson.build:223]: you're right  =)

**Pull request merged**